.env.example

```diff
@@ -15,3 +15,4 @@ S3_BUCKET=s3-bucket
 S3_REGION=s3-region
 S3_CUSTOM_DOMAIN=https://s3-custom-domain-if-it-exists.com
 MAX_UPLOAD_SIZE=size-in-bytes
+GOOGLE_ANALYTICS_CODE=
```

Comment:
Just to keep it consistent with the rest of the file, could you change it to something like `GOOGLE_ANALYTICS_CODE=google-analytics-code`